### DISCUSSION TITLE
Mysql or PostgreSQL  ?

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#8354] [Q&A] [answered]
What better for Orchard Core Mysql or PostgreSQL( running in container) ?

**Comment by Skrypt** [comment, marked as answer]
SQL Server 😉 
I would personally prefer PostgreSQL over MySQL.

**Comment by hishamco** [comment]
I prefer MySQL, because PostgresSQL is sensitive to the casing

**Reply by Skrypt** [reply]
SQL Server can be too.
